openmdao/solvers/ln_gauss_seidel.py

```diff
@@ -216,7 +200,7 @@ def _norm(self, system, mode, rhs_mat):
 
         # we used to build gs_outputs up using dumat, but dumat is already
         # identical to gs_outputs in the vois we care about, so just use it.
-        system.apply_linear(mode, ls_inputs=system._ls_inputs, vois=rhs_mat.keys(),
+        system.sys_apply_linear(mode, ls_inputs=system._ls_inputs, vois=rhs_mat.keys(),
                             gs_outputs=system.dumat)
```

Comment:
spacing, pep-8 cleanup (check all spots where you changed the function name, and tab over the 2nd line)